packages/visual-dev/src/components/Buttons.tsx

```diff
@@ -0,0 +1,352 @@
+import React, { SyntheticEvent, useState } from "react";
+import styled, { keyframes } from "styled-components";
+
+interface PBProps {
+  padding?: string | false;
+  buttonColor?: string | false;
+  darkercolor?: string | false;
+  disabledcolor?: string | false;
```

Comment:
I don't understand why these are `| false` - they are already possibly undefined, so I feel like this is unnecessary notation.